lib/bitcoinerlab.js

```diff
@@ -0,0 +1,152 @@
+const secp256k1 = require('@exodus/bitcoinerlab-secp256k1')
+let elliptic
+
+const verbose = false
+function loadLegacy (method, message) {
+  if (verbose) {
+    if (!message) {
+      console.warn(`${method} is not implemented, using legacy approach...`)
```

Comment:
that's a lot of warnings - one per method invocation, why do we need to warn for these again?

Reply:
Not really required `verbose = false` makes sure we will only see logs when required